types/viewRaidData.d.ts

```diff
@@ -7,7 +7,7 @@ interface UserStats {
   commits: number
 }
 
-interface ViewRaidData {
+interface Raid {
```

Comment:
Why the naming convention change?

Reply:
The convention is what was already there, regardless if you found it superfluous or not. 

As for changing things that is always up for discussion.

Reply:
As above suggested `RaidStats`